### PR TITLE
Return submission IDs and always return isAccepted and isSubmitted

### PR DESCRIPTION
**Pull request by AsirAAlam**
- Return submission IDs
- Return isAccepted and isSubmitted even when isBrief is false

||
|-|
|image|
|image|
|image|
|image|

### CODE REVIEW

**Comment by sd0623** [comment]
LGTM, but I will let the backend to approve this. 
Another thing is that is it after the frontend gets the problems list, the brief submission is called for each problem?

**Comment by AsirAAlam** [comment]
> LGTM, but I will let the backend to approve this. Another thing is that is it after the frontend gets the problems list, the brief submission is called for each problem?

I agree let's have a backend take a look as well. 

The isBrief won't be called separately for every problem. The API takes an array of problemIDs. So it will be called once for all the problems in the table.

**Comment by sd0623** [comment]
> > LGTM, but I will let the backend to approve this. Another thing is that is it after the frontend gets the problems list, the brief submission is called for each problem?
> 
> I agree let's have a backend take a look as well.
> 
> The isBrief won't be called separately for every problem. The API takes an array of problemIDs. So it will be called once for all the problems in the table.

Sounds good! 👍

File: functions/src/submissions.ts [hunk 1]
```diff
@@ -30,24 +30,27 @@ export async function getSubmissions(req: Request, res: Response) {
         )
 
         const docs = await getDocs(queries)
-        docs.forEach((doc) => submissionList.push(doc.data()))
-
-        let problem = {}
-        if (isBrief) {
-          const isSubmitted = submissionList.length > 0
-          let isACed = false
```
Comment (cameroncuster) [inline functions/src/submissions.ts:38]: nice var name haha

**Review by cameroncuster — APPROVED** [review]
LGTM! nice

**Pull request merged**